metrics/run/collect_metrics.go

```diff
@@ -175,10 +191,18 @@ func main() {
 		log.Fatal(err)
 	}
 	inputBucket := gcsClient.Bucket(*inputGcsBucket)
-	inputCtx := storage.NewShardedGCSDatastoreContext(ctx, storage.Bucket{
-		Name:   *inputGcsBucket,
-		Handle: inputBucket,
-	}, nil)
+	var inputCtx storage.Loader
+	if *consolidatedInput {
```

Comment:
Hard to spot the diff between these lines.

    ctxF := storage.NewConsolidatedGCSDatastoreContext
    if *consolidateInput {
        ctxF = storage.NewConsolidatedGCSDatastoreContext
    }
    inputCtx := ctxF(ctx, storage.Bucket { ...